### PR TITLE
chore: add styling and content to footer and header

### PR DESCRIPTION
**Pull request by szumi112**

### CODE REVIEW

**Comment by szumi112** [comment]
image

File: src/app/layout.tsx [hunk 1]
```diff
@@ -26,13 +26,18 @@ export default function RootLayout({
...
-    <Footer />
-    </body>
+      <body
+        className={cn(
+          `${geistSans.variable} ${geistMono.variable}`,
+          "bg-neutral-100 font-sans grid grid-rows-[auto_1fr_auto] items-center justify-items-center min-h-screen  gap-16 sm:p-0"
+        )}
+      >
+        <Header />
+        <main className="flex flex-col gap-8 row-start-2 items-center sm:items-start">
```
Comment (ybtam) [inline src/app/layout.tsx]: i would avoid styling the main component unless it will be used in multiple places.

File: src/app/layout.tsx [hunk 2]
```diff
@@ -26,13 +26,20 @@ export default function RootLayout({
...
-    <Header/>
-    <main className="flex flex-col gap-8 row-start-2 items-center sm:items-start">
-      {children}
-    </main>
-    <Footer />
-    </body>
+      <body
+        className={cn(
+          `${geistSans.variable} ${geistMono.variable}`,
+          'grid grid-rows-[auto_1fr_auto] gap-16',
```
Comment (ybtam) [inline src/app/layout.tsx]: don't use grid unless we need to make a grid, much better to use flex flex-col or flex-wrap

File: src/app/layout.tsx [hunk 3]
```diff
@@ -26,13 +26,20 @@ export default function RootLayout({
...
+          'items-center justify-items-center',
+          "bg-neutral-100 font-sans  min-h-screen  sm:p-0"
+        )}
+      >
+        <Header />
+        <main className="flex flex-col gap-8 row-start-2 items-center sm:items-start">
```
Comment (ybtam) [inline src/app/layout.tsx]: don't style it unless this is the style for the whole project, reason being that if another page wants a different style we will be stuck with it, do this in the page.ts

File: src/components/footer/index.tsx [hunk 4]
```diff
@@ -1,9 +1,16 @@
...
 
 const Footer = () => {
-  return <footer className="row-start-3 flex gap-6 flex-wrap items-center justify-center">
-    <div>Footer</div>
-  </footer>
-    };
+  return (
+    <footer
+      className="text-center py-3 bg-neutral-50 rounded-t-md w-full sm:w-[594px] flex items-center justify-center"
+      style={{ boxShadow: "0 -1px 6px rgba(0, 0, 0, 0.05)" }}
```
Comment (ybtam) [inline src/components/footer/index.tsx]: add custom box shadow with tailwind

**Review by ybtam — CHANGES_REQUESTED** [review]

**Review by szumi112 — COMMENTED** [review]
changes made

**Pull request merged**